social_media_pinterest/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': "Pinterest Social Media Icon Extension",
+    'summary': """Pinterest Extension for the social media icons from the
+    odoo core""",
+    'author': "Diagram Software, S.L."
```

Comment:
add a comma between this and oca. And better don't use commas in your own company name, as the app store splits the authors by comma